### PR TITLE
gallery main page animation

### LINKED ISSUE #138: Gallery Main Page Animation
image

- do the main page only, the other pages are someone else's issues only.

### PR DESCRIPTION
**Pull request by xwRongx**
image
Tried to use different animations but let me know if it's too busy!

### CODE REVIEW

**Comment by seanquiambao** [comment]
wait is that screenshot mid-animation or was the component misaligned on your end?

**Comment by xwRongx** [comment]
> wait is that screenshot mid-animation or was the component misaligned on your end?

image
it was just a badly positioned screenshot lol, it looks centered on my end

**Comment by seanquiambao** [comment]
oh okii sounds good haha

File: src/app/gallery/page.js [hunk 1]
```diff
@@ -45,7 +53,7 @@ const Page = () => {
         </div>
       </div>
 
-      <div className="mb-14">
+      <div className={`${inView && "animate-flip-up animate-once"} mb-14`}>
```
Comment (qhgill) [inline src/app/gallery/page.js]: you may want to add another ref for this section as the user won't see the animation

**Review by qhgill — CHANGES_REQUESTED** [review]
looks good so far! try experimenting with ref placement so animations play when the viewer scrolls instead of all of them playing at once.

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**